### DISCUSSION TITLE
Is it okay to ignore the first two topics?

### DISCUSSION THREAD

**Discussion by lrettich** [MaartenGr/BERTopic#1164] [Q&A] [answered]
_Copy from a [question](https://datascience.stackexchange.com/questions/120680/bertopic-is-it-okay-to-ignore-the-first-two-topics) on datascience.stackexchange.com, where the question is not getting a lot of attention:_ 

I used BERTopic to generate a topic model over a large dataset of texts. The result is very appealing and the modeled topics are mostly perfectly interpretable for a human, especially compared to other topic modeling approaches.

According to the documentation the topic with number -1 refers to outliers and should be ignored. Topic -1 is in my case indeed a topic consisting of unrelated common words without a possible interpretation.

However, in my topic model, as well the second topic with number 0 is just a mixture of unrelated words and not a "nice topic" in terms of human sense. All the following topics are very nice topics with a clear meaning.

My question: Is it okay to ignore the first two topics (-1 & 0) modeled by BERTopic and only start using the topics from topic 1 on? Or is this problematic and indicating an issue with the model? Is there a parameter that can be changed to change this behavior in order to ensure that only topic -1 will be an uninterpretable topic?

**Comment by MaartenGr** [comment, marked as answer]
It might be that topic 0 groups documents together that have little meaning and as such can be considered to be a topic. From that perspective, they could indeed be considered outliers. For example, if you pass the model a couple of hundred of empty documents, it will generate a perfectly valid cluster with no labels. That cluster, could then be ignored. You could, however, merge -1 and 0 together with `.merge_topics` or `.update_topics` to make sure they are indeed -1.

**Reply by lrettich** [reply]
Thank you very much for your answer! 
That is a good point to merge these topics to receive a "normal" model.
